src/Microsoft.Health.Fhir.SqlServer/Features/Storage/TvpRowGeneration/ReferenceSearchParamListRowGenerator.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Schema.Model;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage.TvpRowGeneration
+{
+    ////internal class ReferenceSearchParamListRowGenerator : ITableValuedParameterRowGenerator<IReadOnlyList<ResourceWrapper>, ReferenceSearchParamListRow>
```

Comment:
Should we remove this file given that all code is commented?

Usually, we try to avoid as much as possible merging commented code.